### DISCUSSION TITLE
2024-03-13 Weekly Meeting 150 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#450] [Weekly Meetings]
Weekly Meeting 150 is happening on Wednesday, 13 March, 2024 from 9:00 AM PDT (17:00 UTC) - 10:00 AM PDT (18:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting in the NEW SERVER, in the omi-weekly-meeting channel of the [OMI Discord](https://discord.gg/rrHfT7H3).
We regularly use the # omi-meeting-chat channel for text chat.

### Agenda

**check in**

What has everyone been up to?
- Jesse is getting married!  (in Colorado on March 15th!)

Fun finds!

**housekeeping:**

- last week's notes:  https://github.com/omigroup/omigroup/discussions/449
-- was last week's conversation recorded? where do those raw recordings live? Can we add them to github?
- we have to migrate our fiscal sponsor at W3C
- - conversation: can we do a DAO, as well as a US based NPO (???) as well as a European one? (indiebio is researching the EU options). That should cover all bases. Suggest to do it old school, aka direct, with legal advice, not through a company like OC/OCF
- we would like to start publishing our blogs:
- Blog draft for first article here: https://github.com/omigroup/media/issues/78
- - related, do we want a new omi-experiments channel? it is mentioned in the blog draft as an easy way to test a new working group.
- - Do we want a Matrix server? Otherwise we need to remove that line in the draft, or ask if people would want one.
- - 
- we have two resources, one with the website wordpress omigroup.org and one with w3c which has a wordpress newsletter feature.
- - what was discussed here? Suggest to do all blogs on the omigroup site and then email them via the w3c newsletter. Aim to keep w3c static and point all people to discord/omi website?
- we need to announce our new discord
- announce new chairs and thank the previous
- we need to migrate our bots to the new discord
- we need to set up transcription again in the new discord

**Working group check ins**

glTF
publicity
backlog meeting
Demos and discussion

**Comment by indiebio** [comment]
1. Fun finds: Disney holotile https://za.ign.com/star-wars-tales-from-the-galaxys-edge-1/189080/news/disney-unveils-the-holotile-floor-inching-us-closer-to-a-real-life-holodeck

**Comment by indiebio** [comment]
Virtual Wolds Musuem: in April, we will be launching the Virtual Worlds Museum: Platform of the Week. A video series documenting all of the 3D platforms in our research week after week, pls reach out directly if you want to learn more or contribute. - Keyframe

**Comment by indiebio** [comment]
Open Collective conversation:
avatarjoy: Yes, from Open Collective's article, the Open Collective Foundation is a separate entity from them. So none of this effects projects using Open Collective. 

It DOES effect those projects who applied to be fiscally sponsored by the Open Collective Foundation so they could also have nonprofit status,/perks.

Many months back OMI had decided it wanted to be able to have folks give tax deductible donations or go for grant funding possibilities and switch from being under the broader Open Collective to the Foundation. If that process did happen, then OMI will have to migrate away from Foundation and either back to the broader Open Collective or another nonprofit that uses Open Collective platform similar to how the Foundation was if OMI wants to function as a nonprofit.

If this hasn't been discussed at an OMI meeting it should.

**Reply by indiebio** [reply]
avatarjoy: Happy to chat on US nonprofit stuff. It isn't as fun or as easy requirements wise for a nonprofit versus a for profit of course. There is an older doc when @funksoup and @avatarjoy walked through options last year on all of this. Will refine and share

**Reply by indiebio** [reply]
indiebio: link may be relevant: https://fossfunding.com/
would it make sense to a) do this oldschool (being old...) and get a lawyer and register the NPO and jump through whatever the hoops are for the US. Then, 2) also do that for the EU, and then 3) also have a DAO fo crypto, as jin suggested?

**Reply by indiebio** [reply]
TODO: need to update on the website if donations to us is tax deductable

**Reply by indiebio** [reply]
TODO: edit the https://opencollective.com/omigroup page - do we have 501c6 fiscal sponsorship? If not (we think not) then edit the text to be clear about it.

- [ ]

**Reply by JoyceBettencourt** [reply]
OMI should not mention tax-deductible regarding donations. If we did not move underneath the Open Collective "Foundation," then we have yet to be under the fiscal sponsorship of a 501c3 nonprofit, and being under the regular Open Collective org's 501c6 does not transfer any ability for us to say donations are tax deductible.

Maybe some rewording on OMI's site that: Donations to OMI help support our community and work we do in helping create the open metaverse. Donations are currently not tax-deductible.

**Comment by indiebio** [comment]
Social events - first Tuesday of the month, same time as these meetings (afternoon in the EU). - jimmy.
Discord channel, same as the meetings.
By next week jimmy / publicity team will have a place where people can add suggestions for the format of these events - suggest speakers, demo days, whatever.

**Comment by indiebio** [comment]
from Publicity team: 
Please use this for the table of contents of any blog activity to review :
https://github.com/omigroup/media/discussions/77

**Comment by indiebio** [comment]
Next week is GDC, so some/many won't be able to attend this meeting. Enjoy GDC! https://gdconf.com/

**Comment by indiebio** [comment]
SoloSolipsists shows demo using VisionPro and stuff 
![image](https://github.com/omigroup/omigroup/assets/23474368/327306fe-792c-4861-ab3f-a32f201fed2e)

**Comment by indiebio** [comment]
talk about immersive card, fully animated gltf congratulations card!
